### PR TITLE
did all loops and did transition

### PR DESCRIPTION
**Pull request by Pashahu1**
[DEMO LINK](https://Pashahu1.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.calendar {
+  position: absolute;
+  top: 25%;
+  left: 25%;
```
Comment (vlad-demchuk) [inline src/styles/blocks/calendar.scss]: better use flex and margins to center it

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  &__day {
+    width: $number-size;
+    height: $block-height;
+    @extend %settings-blok-structure;
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number}) {
+        &::before {
+          color: black;
+          line-height: 100px;
```
Comment (vlad-demchuk) [inline src/styles/blocks/calendar.scss]: leave only content: "#{$number}";

no need to apply other styles on each iteration

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          content: "#{$number}";
+        }
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
```
Comment (vlad-demchuk) [inline src/styles/blocks/calendar.scss]: move `cursor: pointer;` to class instead of hover

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
center the catalog
image

File: src/index.html [hunk 4]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--first-day--tue calendar--mounth--lenght-31">
```
Comment (volodymyr-soltys97) [inline src/index.html]: `lenght` - is incorrect. Should be `length`

File: src/styles/blocks/calendar.scss [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  @extend %calendar-settings;
+
+  $mounths:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/calendar.scss]: This should be entered in the variables file

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "./reset/reset.css";
+
+@import "./utilities/variebles";
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss]: `variebles` - is incorrect. Should be `variables`. Rename this file

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Great job, but let's fix some minor bugs to improve your code

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 🔥 
But for the future, create a separate branch for the task

**Pull request closed without merging**